web/.env.mainnet.public

```
@@ -13,4 +13,7 @@ export REACT_APP_SHUTTER_API="https://shutter-api.chiado.staging.shutter.network
 
 
 #subgraph-status-banner
-export REACT_APP_SHOW_STATUS_BANNER=true
\ No newline at end of file
+export REACT_APP_SHOW_STATUS_BANNER=true
+
+#scout
+export REACT_APP_SCOUT_SUBGRAPH_URL=https://indexer.hyperindex.xyz/[api-key]/v1/graphql
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)